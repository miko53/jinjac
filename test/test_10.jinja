TEST BUILD-IN TRUNCATE FUNCTION
truncate simple:
==> {{ 'Ab dsfl smdflk ef fef' | truncate(10) }} Doit donner "Ab dsfl sm"

