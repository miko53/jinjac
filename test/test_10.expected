TEST BUILD-IN TRUNCATE FUNCTION
truncate simple avec argument par default:
==> 'Ab...' Doit donner "Ab..."
==> 'Ab dsfl smdflk ef fef' Doit donner "Ab dsfl smdflk ef fef"
==> 'Ab...' Doit donner "Ab..."
==> 'Ab dsfl...' Doit donner "Ab dsfl..."

